code/modules/mob/living/carbon/human/species.dm

```diff
@@ -66,6 +66,14 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 	var/list/mutant_bodyparts = list()
 	///Internal organs that are unique to this race, like a tail.
 	var/list/mutant_organs = list()
+	///The bodyparts this species uses. assoc of bodypart string - bodypart type. Make sure all the fucking entries are in or I'll skin you alive.
+	var/list/bodypart_overides = list(
```

Comment:
Something something tests

Reply:
oh yeah good idea

Reply:
spelled overrides..

Reply:
they ride

Reply:
i made a test but it runtimes like crazy because of the shitcode that is species runtiming when you create/delete them like what happens in a unit test, also dullahans.